bin/run_neuralnet.sh

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 # EXAMPLE USAGE 
-# bin/run_neuralnet.sh -local true -maxIter 100 -conf ../src/test/resources/configuration/neuralnet -input dolphin-dnn/src/test/resources/data/neuralnet  -timeout 800000
+# bin/run_neuralnet.sh -local true -maxIter 100 -conf dolphin-dnn/src/test/resources/configuration/neuralnet -input dolphin-dnn/src/test/resources/data/neuralnet  -timeout 800000 -logPeriod 1000
```

Comment:
Sure. When you revert the configuration file, please don't forget to remove `logPeriod` option in the example usage because this option is unnecessary with local parameter providers.